core/validation/const.py

```diff
@@ -1,5 +1,10 @@
 import numpy as np
 import pandas as pd
 
+from core.validation.pre_transformation_validation_schema import TF_ROUND_3, TF_ROUND_4
+
 # values we treat as null
 NA_VALUES = {"", np.nan, None, pd.NA}
+
+# reporting round to schema for pre-transformation validation
+REPORTING_ROUND_TO_PRE_TRANSFORMATION_SCHEMA = {3: TF_ROUND_3, 4: TF_ROUND_4}
```

Comment:
That will cause a circular import with this though so this will need to be moved - I think `core/validation/pre_transformation_validation_schema.py` may be best?

Reply:
OK, can move both to core/validation/pre_transformation_validation_schema.py

Reply:
Sounds good